catkin_tools/verbs/catkin_build/build.py

```diff
@@ -732,12 +733,24 @@ def set_error_state(error_state):
                     # Print them in order of started number
                     for job_msg_args in sorted(executing_jobs, key=lambda args: args['number']):
                         msg += clr("[{name} - {run_time}] ").format(**job_msg_args)
-                    msg_rhs = clr("[{0}/{1} Active | {2}/{3} Completed]").format(
-                        len(executing_jobs),
-                        len(executors),
-                        len(packages) if no_deps else len(completed_packages),
-                        total_packages
-                    )
+
+                    if MakeJobServer.get_instance().supported:
```

Comment:
I guess you're assuming here that `get_instance()` was already called, otherwise it will call the constructor with no args. Could that be a problem? Should there be a function to get and instance only if it already exists, otherwise fail?